semantic_conventions/logs/media.yaml

```
@@ -0,0 +1,49 @@
+groups:
+  - id: log
+    prefix: log
+    type: attribute_group
+    brief: "Describes Log attributes"
+    attributes:
+      - id: iostream
+        requirement_level: opt_in
+        brief: >
+          The stream associated with the log. SHOULD be one of: `stdout`, `stderr`.
+        type:
+          allow_custom_values: false
+          members:
+            - id: stdout
+              value: 'stdout'
+              brief: 'Logs from stdout stream'
+            - id: stderr
+              value: 'stderr'
+              brief: 'Events from stderr stream'
+  - id: log.file
```

```suggestion
  - id: attributes.log.file
```

So it's clear this is an attribute group. Similar as in https://github.com/ChrsMark/semantic-conventions/blob/main/semantic_conventions/http-common.yaml#L2

---

Addressed in the latest commit: [90fc179](https://github.com/open-telemetry/semantic-conventions/pull/133/commits/90fc179399568760a0207c303133f1155a670db4)